src/react-imgix-bg.jsx

```diff
@@ -46,7 +47,17 @@ class BackgroundImpl extends React.Component {
       return true;
     }
 
-    return false;
+    // If we made it here, we need to check if the "top-level"
+    // props have changed (e.g. disableLibraryParam).
+    const shallowPropsEqual = shallowEqual(this.props, nextProps);
+
+    // We also need to check the imgixParams.
+    const shallowParamsEqual = shallowEqual(
+      this.props.imgixParams,
+      nextProps.imgixParams
+    );
+
+    return shallowPropsEqual && shallowParamsEqual;
```

Comment:
This is great so far! Two things that should be done here:

1. `imgixParams` shouldn't be checked on L52
2. `htmlAtrtibutes` should also be shallowly compared.

This could be done like this:
```jsx
    const customizer = (value, other, key) => {
        if (key === "imgixParams" || key === "htmlAttributes") {
            return shallowEqual(value, other);
        }
        return undefined; // Use default behaviour
    };

    // If we made it here, we need to check if the "top-level"
    // props have changed (e.g. disableLibraryParam).
    const shallowPropsEqual = shallowEqual(this.props, nextProps, customizer);

    return shallowPropsEqual;
```